app/src/main/res/layout/activity_camera.xml

```diff
@@ -268,5 +268,6 @@
         layout="@layout/element_bottom_navigation"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="0dp"/>
+        android:layout_marginBottom="0dp"
+        android:visibility="gone"/>
```

Comment:
@srv-twry Don't hide the bottom navigation bar. What I meant was that the camera click button and the seek bars could be on top of the preview instead of being in a separate bar at the bottom.
Something like this :
![image](https://user-images.githubusercontent.com/22395998/33760871-4c30e3a2-dc2d-11e7-9d3e-700b6d891d6e.png)

Reply:
@mayank8318 Is it fine ?
![](https://user-images.githubusercontent.com/21277837/33790633-e9de47c8-dca7-11e7-965b-2336047e01e2.png)

Reply:
Yes :+1: